source/process/mobile-release-process.md

```diff
@@ -0,0 +1,61 @@
+# Mobile App Release Process
+
+The Mattermost Mobile Apps team works on a monthly release process, with a new version submitted to the App Store and Google Play Store on the 5th of each month. Note: App Store approval may take a few days after the 5th. 
+
+## Release Timeline
+
+Notes:
+
+- All cut-off dates are based on 10am (San Francisco Time) on the day stated.
+- T-minus counts are measured in "working days" (weekdays other than major holidays in the US and Canada) prior to release day.
+
+### A. (T-minus 8 working days) Code Complete and Release Candidate Cut 
+
+1. PM:
+  - Submit pull request for changelog
+  - Check the minimum version number and submit pull request to update 
+2. Dev:
+  - Finish review of last pull requests for the current release and merge
+3. QA:
+  - Confirm all pull requests merged into the current release have been tested
+4. Build:
+  - Cut release candidate build
+
+
+### B. (T-minus 7 working days) Release Candidate Testing
+
+1. QA:
+  - Post release testing instructions and spreadsheet to Native Mobile Apps channel
+2. Team:
+  - Test assigned areas of the testing spreadsheet and file any bugs found in Jira 
+  - Daily triage of hotfix candidates and decide on whether and when to cut next RC or final
+3. Dev:
+  - Make pull requests for hotfixes to the release branch
+  - Review PRs made to release branch and merge changes into both the release branch and master
```

Comment:
I would exclude master from this, then we can merge the changes in the release branch back to master